### DISCUSSION TITLE
How To Use Table configure Function To Configure Desired Ordering

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#14259] [General]
Hey, I am seeing that there is a configure option for a Netbox Table, but am looking for a good guide on how I can implement it, like the Plugin Demo. I am trying to see if I can use it n a plugin to specify which fields to show to the user by default and which arent shown. If this isnt possible, please let me know. If it is possible, is there a good description available?

**Comment by jeremystretch** [comment]
The ability to configure table columns is a user-specific feature. It allows each user to customize the display and ordering of each table to his or her own preferences. You can specify which columns to show by default by setting `default_columns` under the table's `Meta` class.

**Reply by mr1716** [reply]
So, that means the default_columns is user based. What happens if I don’t set that value? 

Is there a way to have Netbox specify fields A, C, and D as default for instance 1 of table X, but have that same table specify fields B, D, and E in a 2nd instance of table X?
